src/Components/Admin/Rooms/roomCard.js

```diff
@@ -53,94 +122,119 @@ export default function RoomCard({ data, handleEditClick, handleDeleteClick }) {
         setAnchorEl(null);
     };
 
-    const handleCardClick = () => {
+    const handleDoubleClick = () => {
         history.push(`/admin/environment/${data.id}`);
     };
 
+    const handleCardClick = () => {
+        setIsHighlighted(true);
+    };
+
+    const handleClickAway = () => {
+        setIsHighlighted(false);
+    };
+
     const open = Boolean(anchorEl);
 
     return (
-        <Grid
-            container
-            item
-            xs={12}
-            md={6}
-            lg={4}
-            spacing={1}
-            alignItems="center"
-            justify="flex-start"
-            className={classes.card}
-        >
+        <ClickAwayListener onClickAway={handleClickAway}>
```

Comment:
Genuinely never used the `ClickAwayListener` library before... nice find!

Reply:
Yeah +1!